wireless/bluetooth/Kconfig

```diff
@@ -165,6 +165,16 @@ config BLUETOOTH_BUFFER_IRQRESERVE
 		interrupt level.  This setting only needs to be non-zero if your
 		low-level Bluetooth driver needs to do such allocations.
 
+config BLUETOOTH_CNTRL_HOST_FLOW_ENABLE
+	bool "Enable Controller to Host Flow Control"
+	default n if BT_CTRL_HOST_FLOW_CONTROL_SUPPORT_DISABLED
```

Comment:
this way you still rely on the arch specific flag.
You should `select BLUETOOTH_CNTRL_HOST_FLOW_ENABLE` in your arch specific `Kconfig` when BLE is enabled.

Reply:
````
config ESP32_BLE
	bool "BLE"
	default n
	select ESP32_WIRELESS
	select BLUETOOTH_TXCMD_PINNED_TO_CORE if SMP
        select BLUETOOTH_CNTRL_HOST_FLOW_ENABLE if WIRELESS_BLUETOOTH
```

Reply:
Good point @raiden00pl !!! @robertc2000 actually that flag BT_CTRL_HOST_FLOW_CONTROL_SUPPORT_DISABLED should be defined wireless/bluetooth/Kconfig as disabled by default and at arch (like ESP32 Kconfig) that flag should be enabled

Reply:
Actually, for ESP32, the flow should be disabled, so BLUETOOTH_CNTRL_HOST_FLOW_ENABLE should be off only for it.

I agree though, that's a good point! Thanks for the hint! Pushed the changes now which correct that.